build.gradle

```diff
@@ -249,37 +243,55 @@ dependencies {
         implementation 'org.apache.commons:commons-collections4:4.4'
         implementation 'org.openapitools:jackson-databind-nullable:0.2.2'
     } else {
-        implementation ('bio.terra:terra-common-lib:0.0.89-SNAPSHOT') {
-            exclude group: 'org.broadinstitute.dsde.workbench', module: 'sam-client_2.12'
-        }
+        implementation 'bio.terra:terra-common-lib:0.1.10-SNAPSHOT'
     }
 
-    implementation 'bio.terra:terra-policy-client:1.0.4-SNAPSHOT'
-    implementation 'bio.terra:terra-resource-buffer-client:0.4.3-SNAPSHOT'
-    implementation 'bio.terra:externalcreds-client-resttemplate:0.72.0-SNAPSHOT'
-    implementation 'org.glassfish.jersey.inject:jersey-hk2:2.30.1'
+    implementation 'org.broadinstitute.dsde.workbench:sam-client_2.13:0.1-a91acd0'
+    implementation 'bio.terra:terra-policy-client:1.0.11-SNAPSHOT'
+    implementation 'bio.terra:terra-resource-buffer-client:0.198.42-SNAPSHOT'
+    implementation 'bio.terra:externalcreds-client-resttemplate:1.3.0-SNAPSHOT'
```

Comment:
I'll flag that this latest version of ECM client includes this change:

https://github.com/DataBiosphere/terra-external-credentials-manager/pull/141

Which means that we can remove this [2 year old stopgap](https://github.com/DataBiosphere/jade-data-repo/blame/3198570b711a39510684a03b0cd685756d839b6b/src/main/java/bio/terra/service/auth/ras/EcmService.java#L102-L107) (a unit test in [EcmServiceTest](https://github.com/DataBiosphere/jade-data-repo/blob/3fbca34adf4bba5a056da7df7bedb7cc7b5f4a77/src/test/java/bio/terra/service/auth/iam/ras/EcmServiceTest.java#L53) would need to be updated):

```
      String passport = oidcApiService.getOidcApi(userReq).getProviderPassport(RAS_PROVIDER);
      // Passports returned by OidcApi have a bug in their formatting:
      // double quotes must be stripped if passing back to PassportApi for validation,
      // otherwise the passport will not be considered valid JWT.
      // This stopgap can be removed when the client is fixed:
      // https://broadworkbench.atlassian.net/browse/ID-128
      return StringUtils.strip(passport, "\"");
```

But, the stopgap is both forwards and backwards-compatible, so we can also ticket its removal and handle it separately if you'd prefer.